project_templates/roundtable_aiohttp_bot/{{cookiecutter.name}}/.github/workflows/ci.yaml

```diff
@@ -20,6 +20,9 @@ jobs:
         with:
           python-version: {{ "${{ matrix.python }}" }}
 
+      - name: Run pre-commit
+        uses: pre-commit/action@v2.0.0
```

Comment:
Do you think it'd be better to let the minor or patch versions float? I know we've gotten unnecessary noise in the past from action updates.

Reply:
Unfortunately, the pre-commit/action repository doesn't have major version tags the way that the other action repositories do. They only tag with their full version strings, and the `@` label isn't magical (at least so far as I understand it). It still has to match a tag.